README.md

```diff
@@ -1,6 +1,6 @@
 # Django Rest Password Reset
 
-[![PyPI version](https://badge.fury.io/py/django-rest-passwordreset.svg)](https://badge.fury.io/py/django-rest-passwordreset)
+[![PyPI version](https://img.shields.io/pypi/v/django-rest-passwordreset.svg)](https://img.shields.io/pypi/v/django-rest-passwordreset)
```

Comment:
This currently links to the image. Please link it to pypi: https://pypi.org/project/django-rest-passwordreset/

Reply:
Done! =)